.github/workflows/test.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        python-version: [3.8, 3.9, 3.10, 3.11]
+        python-version: [3.8, 3.9, 3.10.10, 3.11]
```

Comment:
Seems strange but it works

Reply:
Github convert 3.10 to 3.1, so 3.10 doesn't work (probably 3.10.10 converted to 3.10.1 bl bl)